content/source/docs/enterprise/api/team-access.html.md

```diff
@@ -73,15 +71,20 @@ $ curl \
 
 ## Add Team access to a Workspace
 
-| Method | Path           |
-| :----- | :------------- |
-| POST | /team-workspaces |
+`POST /team-workspaces`
 
-### Parameters
+### Request Body
 
-- `access` (`string: <required>`) - `read`, `write`, or `admin`
-- `team` (`string: <required>`) - The ID of the team to add to the workspace.
-- `workspace` (`string: <required>`) - The workspace ID to which the team is to be added.
+This POST endpoint requires a JSON object with the following properties as a request payload. (Properties without a default value are required.)
```

Comment:
Could the default statement here become more prominent? Could we denote it visually too in the table?

Reply:
Like, just put that sentence on a separate line? I guess. 

Putting it in the table seems redundant and noisy, though.

Reply:
Yeah, a new line, maybe bold?